__tests__/endpoints/app.test.js

```diff
@@ -105,3 +105,28 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200 and returns an array of objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+          });
+          expect(typeof article.comment_count).toBe("number");
+        });
+        expect(articles.length).toBe(articleData.length);
```

Comment:
I like the idea of this but there isnt a need to go this far to check the length is correct. The idea here should be to ensure the array contains at least 1 thing and then the above forEach will ensure that they look as we expect them to. A nicer more flexible alternative to this would be to use .toBeGreaterThan()